### ISSUE TITLE
Update documentation for NUOPC_CompAttributeIngest using freeformat with lists

### ISSUE THREAD

**Issue by danrosen25** [esmf-org/esmf#333]
Update documentation for NUOPC_CompAttributeIngest to acknowledge that whitespace delimited lists do not work.
Lists will fail (quietly) if line has spaces.

see issue #293

**Comment by danrosen25** [comment]
@theurich @anntsay 

Does this change read okay?

Replacing
```
-!   Important: Attributes ingested by this method are stored as type character
-!   strings, and must be accessed accordingly. Conversion from string into a
-!   different data type, e.g. {\tt integer} or {\tt real}, is the user's
-!   responsibility.
```
with
```
+!   Important: Attributes ingested by this method are stored as type character
+!   strings, and must be accessed accordingly. Conversion from string into a
+!   different data type, e.g. {\tt integer} or {\tt real}, is the user's
+!   responsibility. This method does not support value lists. Attribute values
+!   ingested by this method must not contain whitespace within the value. If
+!   whitespace is found within the value the attribute will not be added to
+!   the comp.
+!
+!   The following entry will be ignored
+!   \begin{verbatim}
+!     attributeName = attributeValue1 attributeValue2 attributedValue3
+!   \end{verbatim}
+!   The following entry is acceptable
+!   \begin{verbatim}
+!     attributeName = attributeValue1,attributeValue2,attributedValue3
+!   \end{verbatim}
```

**Comment by theurich** [comment]
Dan, I think that could do. I was hoping though that we could use the opportunity to more clearly lay out the formatting rules in general with respect to white spaces. I know that will require a lot more additions to explain it.

**Comment by theurich** [comment]
@danrosen25 do you think you will want to push something in today still?

**Comment by danrosen25** [comment]
> @danrosen25 do you think you will want to push something in today still?

Sorry, I lost track with the ESMF performance test. I have a PR open now. I think it's good to go but I wasn't sure what you want for 'more clarity' but I tried in the PR.

**Comment by theurich** [comment]
I just merged PR #339 into release/8.8.0.
